addon/components/denali-navbar/menu.hbs

```diff
@@ -0,0 +1,5 @@
+<DenaliNavbar::Item ...attributes>
```

Comment:
Does this produce the same structure as shown here: https://denali-design.github.io/denali-css/?path=/story/components-navbar-elements--menu

Reply:
```
            <div class="menu menu--nav is-right">
                <div class="nav-item menu-trigger"
                    >Menu<i class="d-icon d-arrowhead-down is-small"></i
                ></div>
                <div class="menu-content">
                    <a>Menu Item #1</a>
                    <a>Menu Item #2</a>
                    <a>Menu Item #3</a>
                </div>
            </div>
```

Reply:
Yeah @jkusa 
Updated the navbar section template with denail-menu to render as mentioned above

Reply:
https://github.com/denali-design/denali-ember/blob/a26e2d6700ad0136c092e619cde51bce0a84c4d0/addon/components/denali-navbar/section.hbs#L11

Reply:
Try running the addon locally and edit the `application.hbs` with this

```hbs
<Right.Menu as |Menu|>
  <Menu.Trigger>Hi</Menu.Trigger>
  <Menu.Content>ok</Menu.Content>
</Right.Menu>
```

It needs to match the above html so that it shows up correctly

Reply:
I Updated the application template and run it
menu-trigger is inline element (span tag) 
Rest everything is rendering as expected